JourneysSquad/JourneysSquad/View/AddBillView/Subviews/LabelWithTextFieldView.swift

```diff
@@ -0,0 +1,101 @@
+import SwiftUI
+
+/// A text field that allows to enter and edit text and has a label above itself.
+struct LabelWithTextFieldView: View {
+    // MARK: - Internal interface
+
+    /// The text entered into the field.
+    @Binding var input: String
+
+    /// A text displayed in the label.
+    let labelText: String
+
+    /// A placeholder text displayed within the field.
+    let placeholder: String
```

Comment:
This is not a placeholder, this is a text for placeholder, so you have to name it:
```suggestion
    let placeholderText: String
```

You haven't follow [my advice](https://github.com/ios-course/link-team-project/pull/51#discussion_r1212662943) and haven't double-check the PR, so as we have discussed I'm not going to review your later PRs anymore (in the current sprint).

Reply:
Fixed